lib/rexml/text.rb

```diff
@@ -269,14 +269,16 @@ def wrap(string, width, addnewline=false)
     end
 
     def indent_text(string, level=1, style="\t", indentfirstline=true)
+      Kernel.warn("#{self.class.name}#indent_text is deprecated. See REXML::Formatters", uplevel: 1)
       return string if level < 0
-      new_string = ''
+
+      new_string = +''
       string.each_line { |line|
         indent_string = style * level
         new_line = (indent_string + line).sub(/[\s]+$/,'')
         new_string << new_line
       }
-      new_string.strip! unless indentfirstline
+      new_string.strip! unless indentfirstline # no change
```

Comment:
```suggestion
      new_string.strip! unless indentfirstline
```

Reply:
I am very confused.  I am certain I removed that...